### PR TITLE
Responsive statistics cards (EXPOSUREAPP-4700)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Implement responsive statistics cards
- Unify code and layout style in all cards

### CODE REVIEW

**Comment by AlexanderAlferov** [comment]
Code looks great.
Tested on Samsung S8. Overall its good:

Vuzix View_20210121_171642_603


But **7-Tage-Inzidenz** have some issues with max size font:
Screenshot 2021-01-21 at 17 19 36

**Review by harambasicluka — CHANGES_REQUESTED** [review]
Agree with @AlexanderAlferov, tested it on two devices looks relay good except for the 7-day-incidence card. (haven't done a code review)

image

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsLayoutManager.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package de.rki.coronawarnapp.statistics.ui.homecards
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class StatisticsLayoutManager : LinearLayoutManager {
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsLayoutManager.kt:9]: Could you please add a comment above why this is needed.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml [hunk 2]
```diff
@@ -4,147 +4,138 @@
...
+            app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
+            android:id="@+id/footnote"
             style="@style/StatisticsCardValueLabel"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/card_padding"
             android:layout_marginBottom="@dimen/spacing_small"
+
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml:104]: unnecessary line

**Review by harambasicluka — APPROVED** [review]
Latest commit fixed the broken layout on Samsung devices with a very large font.

Marked two minor things.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsLayoutManager.kt (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsLayoutManager.kt:13]: yes

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml [hunk 3]
```diff
@@ -4,138 +4,126 @@
...
         <ImageButton
-            android:id="@+id/statistics_card_info_button"
+            android:id="@+id/info_statistics"
             style="@style/StatisticsCardInfoButton"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
+        <androidx.constraintlayout.widget.Guideline
```
Comment (BMItter) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:97]: Usage of Barrier/guideline within ConstraintLayouts is good to see 👌

**Review by BMItr — APPROVED** [review]
lgtm, tested on Samsung S10+  👌

**Pull request merged**